src/DynamoCoreWpf/VisualizationManager/VisualizationManager.cs

```diff
@@ -360,6 +359,7 @@ private void WorkspaceRemoved(WorkspaceModel model)
         private void NodeRemovedFromHomeWorkspace(NodeModel node)
         {
             node.PropertyChanged -= NodePropertyChanged;
+            OnNodeDeletedFromWorkspace(node);
```

Comment:
@ramramps Update these names based on the underlying event renamings suggested above.